chunky/src/java/se/llbit/chunky/ui/render/GeneralTab.java

```diff
@@ -205,14 +205,14 @@ public GeneralTab() throws IOException {
     canvasSize.setEditable(true);
     canvasSize.getItems().addAll("400x400", "1024x768", "960x540", "1920x1080");
     canvasSize.valueProperty().addListener(canvasSizeListener);
-    yMax.setRange(0, 256);
+    yMax.setRange(-64, 320);
```

Comment:
You could access the scene and to `this.scene.world.getVersionId() >= 2694` here.

Reply:
ah! will do

Reply:
oh you're right, it doesn't. Maybe lin 300 is a better place to add this?